### DISCUSSION TITLE
What is this number on the overlay?

### DISCUSSION THREAD

**Discussion by kruggles482** [Lailloken/Exile-UI#225] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7832937/230677209-d2af1c8f-113f-4a8f-982e-f663b0b20d1d.png)

**Comment by Lailloken** [comment, marked as answer]
it's from the gear-tracker in the leveling guide. It shows you how many items/gems you can equip (level-wise). Right-clicking the guide-button (book) will open the gear-tracker and those items will be highlighted in green. Right-clicking an item will remove it from the list.
EDIT: it was originally designed around twink-leveling but it can be used in different ways.

**Reply by kruggles482** [reply]
Thanks, King!
